### DISCUSSION TITLE
Do I need mason-lspconfig plugin?

### DISCUSSION THREAD

**Discussion by rudenkornk** [NvChad/NvChad#1878] [Q&A] [answered]
mason-lspconfig is quite popular amongst neovim presets. It is used in `LazyVim`, `LunarVim`, `AstroNvim`, `doom-nvim`, but not in `NvChad`.
I know that there is some support from `NvChad` like `MasonInstallAll` command, and it looks like LSP servers are mostly just working (when listed in both `mason` and `lsp` configs).
So do I even need this plugin? Will it interfere with `NvChad` if I install it?

**Comment by siduck** [comment, marked as answer]
@rudenkornk  it wont interfere, check our example_config repo's featurefull branch. it uses that plugin

**Reply by rudenkornk** [reply]
Thanks!
